packages/forms/src/elements/tiles/Tiles.tsx

```diff
@@ -23,13 +23,13 @@ import { Icon } from './components/Icon';
 import { Label } from './components/Label';
 
 interface ITilesProps extends HTMLAttributes<HTMLDivElement> {
-  /** Value of the selected radio button */
+  /** Sets the value of the selected radio button */
   value?: string;
-  /** Callback when a radio is selected */
+  /** Handles radio selection */
   onChange?: ChangeEventHandler<HTMLInputElement>;
-  /** The name used to reference the value of the control. */
+  /** Sets the name used to reference the value of the control */
   name: string;
-  /** Displays the tiles in their vertical arrangement */
+  /** Arranges the tiles vertically */
```

Comment:
I think there is more helpful language that can be borrowed from here: https://garden.zendesk.com/components/tiles#layout. This is more about the content within the tile, rather than the layout of multiple tiles.